MANIFEST.in

```diff
@@ -21,9 +24,17 @@ include binder/requirements.txt
 include CITATION.cff
 include CODEOWNERS
 
-global-exclude _build build _authors.rst _version.py
+global-exclude _build build _authors.rst
 global-exclude *.o *.py[cod]
 
-# Only for editable installs
 prune docs/api
+
+# The following directory is only used to get the version right for
+# editable installations.
+
 prune src/plasmapy/_dev
+
+# The following file needs to be in the sdist so that the version
+# appears correctly on Read the Docs.
+
+include src/plasmapy/_version.py
```

Comment:
We probably don't need this in here, but explicit is better than implicit, and the comment helps.